proton-j/src/main/java/org/apache/qpid/proton/engine/impl/EventImpl.java

```diff
@@ -217,6 +101,19 @@ public void dispatch(Handler handler)
             dispatch(children.next());
         }
     }
+    
+    @Override
+    public void redispatch(EventType as_type, Handler handler) throws HandlerException 
```

Comment:
Ok. Sounds like you might be using this in a slightly different way than I was thinking, which is likely because I haven't actually used any of the reactor (or handler) stuff at all yet :)

It would be a good idea for some other folks to look this over who actually have, like @rhs and @prestona.